Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -886,7 +928,11 @@ UpdateFspConfig (
   FspsConfig->HDAudioClkGate              = 1;
   FspsConfig->HdAudioDspUaaCompliance     = 1;
 
-  FspsConfig->GraphicsConfigPtr = PcdGet32 (PcdGraphicsVbtAddress);
+  if (PcdGetBool (PcdFramebufferInitEnabled)) {
```

Comment:
should it be PcdGetBool (PcdIntelGfxEnabled) here?

Reply:
GFX device enable is to determin if the PCI device should be visible on PCI bus.   FramebufferInit is to determine if the GFX should be initialized by FSP.